File: second_try/readability2/readability.c
#include <stdio.h>
#include <cs50.h>
#include <string.h>
#include <ctype.h>
#include <math.h>

int main(void)
{
    string text = get_string("Text: ");
    int letters = 0;
    int words = 1;
    int sentences = 0;
    float L = 0;
    float S = 0;
    int len = strlen(text);
    for(int i = 0; i < len; i++)
    {
        if(isalpha(text[i]))
        {
            letters += 1;
        }
        else if(text[i] == ' ')
        {
            words += 1;
        }
        else if(text[i] == '.' || text[i] == '?' || text[i] == '!')
        {
            sentences += 1;
        }
    }
    L = ((float)letters / words) * 100;
    S = ((float)sentences / words) * 100;
    int index = round(0.0588 * L - 0.296 * S - 15.8);
    if(index < 1)
    {
        printf("Before Grade 1\n");
    }
    else if(index > 16)
    {
        printf("Grade 16+\n");
    }
    else
    {
        printf("Grade %d\n", index);
    }
}

File: 105656458-main/python/readability.py
text = input("Text: ")
sentences = 0
words = 1
letters = 0

for i in range(len(text)):
    if text[i].isalpha():
        letters += 1
    elif text[i] == ' ':
        words += 1
    elif text[i] in ('.', '!', '?'):
        sentences += 1

L = letters / words * 100
S = sentences / words * 100
index = round(0.0588 * L - 0.296 * S - 15.8)
print(letters, sentences)

if index < 1:
    print("Before Grade 1")
elif index > 16:
    print("Grade 16+")
else:
    print(f"Grade {index}")





